packs/spell-effects/spell-effect-animal-feature.json

```diff
@@ -0,0 +1,170 @@
+{
+    "_id": "Dcva6SCHr9vWE7nJ",
+    "img": "icons/commodities/biological/eye-lizard-orange.webp",
+    "name": "Spell Effect: Animal Feature",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Animal Feature]</p>\n<p>Without fully transforming your body, you gain one animalistic feature.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 2
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Player Core"
+        },
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.SpecificRule.SpellEffectAnimalFeature.CatEyes",
+                        "value": "cat-eyes"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.SpellEffectAnimalFeature.Claws",
```

Comment:
Could use "PF2E.BattleForm.Attack.Claw" instead, but made it plural to match the text.

Reply:
`You gain a claw attack` ?

Make it claw. Even if Paizo made a typo we know it's in their style guide to be singular there.